blivet-gui.spec

```diff
@@ -63,6 +63,7 @@ fi
 %{_bindir}/blivet-gui
 %{_bindir}/blivet-gui-daemon
 %{_localstatedir}/log/blivet-gui
+/etc/libreport/events.d/blivet-gui_event.conf
```

Comment:
This should use the macro for ``/etc`` and *libreport* should be added to requires.